src/components/dialog/dialog.spec.js

```diff
@@ -264,16 +265,37 @@ describe('$mdDialog', function() {
           ok: 'Next',
           cancel: 'Back',
           title: 'Which Way ',
-          content: '<my-content class="mine">Choose</my-content>'
+          htmlContent: '<my-content class="mine">Choose</my-content> breakfast'
```

Comment:
`class="mine"` seems unnecessary now and `breakfast` doesn't go well with the title...